pages/clubs.jsx

```diff
@@ -231,6 +241,9 @@ var validateClub = function(clubState) {
   if (!clubState.location) {
     errors.push("You must provide a location for your club.");
   }
+  if (clubState.website && !/\./.test(clubState.website)) {
```

Comment:
Note that this is super basic validation; we're essentially just testing to make sure that a `.` appears somewhere in what they've typed. It's way less rigorous than [what Django uses](http://stackoverflow.com/a/7160778), which means that technically, users could submit a URL that our front-end accepts but the teach-api backend doesn't (in which case the user will just get a cryptic message saying that something went wrong and  to retry later).